packages/baseclient/package.json

```diff
@@ -63,18 +63,19 @@
     "start-shogun2": "webpack-dev-server --config config/webpack.shogun2.config.js --hot",
     "build:static": "node scripts/build.js",
     "build:shogun2": "node scripts/build.js",
-    "test": "node scripts/test.js --env=jsdom --config config/jest/jest.config.js",
+    "test": "node scripts/test.js --env=jsdom --config config/jest/jest.config.js --runInBand",
     "lint": "tslint --project tsconfig.json --config tslint.json 'src/**/*.ts' && tsc --noEmit --project tsconfig.json",
-    "pretest": "npm run lint"
+    "pretest": "npm run lint && if [ $CI = false ]; then npm run build:static; fi && CI=false"
```

Comment:
Hmm will this variable CI be visible elsewhere?

Reply:
yep, but i changed this to only be falsy while running production build, should be ok now